src/index.template.html

```diff
@@ -1,17 +1,33 @@
 <!DOCTYPE html>
-<html <% if (htmlWebpackPlugin.options.spa) { %>lang="en"<% } else { %>lang="{{ htmlLang }}" data-vue-meta-server-rendered {{{ meta.inject().htmlAttrs.text() }}}<% } %>>
-<head>
-  <meta charset="utf-8">
-  <% if (!htmlWebpackPlugin.options.spa) { %>{{{ meta.inject().meta.text() }}} {{{ meta.inject().title.text() }}} {{{ meta.inject().link.text() }}} {{{ meta.inject().style.text() }}} {{{ meta.inject().script.text() }}} {{{ meta.inject().noscript.text() }}}<% } else { %><title>vue-starter</title><link rel="shortcut icon" href="/favicon.png" type="image/png" /><link rel="icon" href="/favicon.png" type="image/png" /><% } %>
-  <meta name="mobile-web-app-capable" content="yes">
-  <meta name="viewport" content="width=device-width, initial-scale=1, minimal-ui">
-  <meta name="theme-color" content="#17BEBB"><% for (var chunk of webpack.chunks) {
-  for (var file of chunk.files) {
-  if (file.match(/\.(js|css)$/)) { %>
-  <link rel="<%= chunk.initial?'preload':'prefetch' %>" href="<%= htmlWebpackPlugin.files.publicPath + file %>" as="<%= file.match(/\.css$/)?'style':'script' %>"><% }}} %>
-  <link rel="manifest" href="/manifest.json">
-</head>
-<body>
-<% if (htmlWebpackPlugin.options.spa) { %><div id="app"></div><% } else { %><!--vue-ssr-outlet--><% } %>
-</body>
+<html <% if (htmlWebpackPlugin.options.spa) { %>
```

Comment:
this file should not be formatted, this can lead to weird behaviour with the vue-server-renderer.

Reply:
Good to know! was wondering!